docs/pages/guides/using-analytics.mdx

```diff
@@ -43,6 +44,12 @@ The following list provides common analytics providers that are available in the
 
 <BoxLink
   title="Aptabase"
-  description="Learn how to integrate Aptabase Analytics in your project."
+  description={
+    <>
+      Learn how to integrate Aptabase Analytics in your project.
+      <br />
+      <FOOTNOTE tag="span" theme="success">Works with Expo Go</FOOTNOTE>
+    </>
+  }
```

Comment:
```suggestion
  description="Learn how to integrate Aptabase Analytics into your project. Works with Expo Go."
```

I'd rather have this info in the description instead of adding a footnote.

Reply:
Updated